tests/setup-softhsm.sh

```diff
@@ -107,7 +122,7 @@ CACRTN="caCert"
 let "SERIAL+=1"
 pkcs11-tool --keypairgen --key-type="RSA:2048" --login --pin=$PINVALUE --module="$P11LIB" \
 	--label="${CACRTN}" --id="$KEYID"
-certtool --generate-self-signed --outfile="${CACRT}.crt" --template=${TMPPDIR}/cert.cfg \
+"$certtool" --generate-self-signed --outfile="${CACRT}.crt" --template=${TMPPDIR}/cert.cfg \
```

Comment:
Why?

Reply:
Well consistency for one, we use that form for all other variables.
I also do not quite get why it is quoted now

Reply:
Well, it's quoted because it might contain spaces, but mostly it's quoted because my editor automatically runs shellcheck on bash scripts and shouts at me.

I'll change the variable style.